PC/msvcrtmodule.c

```diff
@@ -167,15 +167,7 @@ static long
 msvcrt_open_osfhandle_impl(PyObject *module, intptr_t handle, int flags)
```

Comment:
The clinic definition could be updated here, e.g. to use a `HANDLE` type based on `void *`:

    class HANDLE_converter(CConverter):
        type = 'void *'
        format_unit = '"_Py_PARSE_INTPTR"'

    class HANDLE_return_converter(CReturnConverter):
        type = 'void *'

        def render(self, function, data):
            self.declare(data)
            self.err_occurred_if("_return_value == INVALID_HANDLE_VALUE", data)
            data.return_conversion.append(
                'return_value = PyLong_FromVoidPtr(_return_value);\n')

Then change the `open_osfhandle` parameter to `handle: HANDLE` and the `get_osfhandle` return type to `-> HANDLE`.